clients/rust/Cargo.toml

```diff
@@ -33,10 +33,8 @@ byteorder = "1.1.0"
 serde_derive = "1.0.*"
 serde = "1.0.*"
 serde_json = "1.0.*"
-
-[dependencies.error-chain]
-version = "0.11.0"
-default-features = false
+failure="0.1"
+log="0.4"
```

Comment:
This doesn't look like it's actually used in the library itself, just the example?